Kernel/StackWriter.h

```diff
@@ -0,0 +1,55 @@
+#ifndef STACKWRITER_H
+#define STACKWRITER_H
+
+#include <AK/StdLibExtraDetails.h>
+#include <AK/Types.h>
+
+namespace Kernel {
+
+class StackWriter {
+    FlatPtr m_top;
+
+public:
+    explicit StackWriter(FlatPtr top) noexcept
```

Comment:
no need to `noexcept`
Serenity does not use exceptions and they are disabled in most cases